spec/models/user/academics/term_registration_spec.rb

```diff
@@ -0,0 +1,299 @@
+require 'spec_helper'
+
+RSpec.describe User::Academics::TermRegistration, type: :model do
+  let(:tuition_calculated) do
+    User::Academics::StudentAttribute.new(tuition_calculated: true)
+  end
+
+  let(:registered) {
+    User::Academics::StudentAttribute.new(registered: true)
+  }
+
+  let(:student_attributes) {
+    instance_double("User::Academics::StudentAttributes")
+  }
+
+  let(:twenty_percent_cnp_exception) {
+    User::Academics::StudentAttribute.new(twenty_percent_cnp_exception: true)
+  }
+
+  let(:term) {
+    instance_double("User::Academics::Term").tap do |mock|
+      allow(mock).to receive(:term_id).and_return("BOGUS-TERM-ID")
+    end
```

Comment:
@developish - I like how you define a double with mocked methods here. Cleaner than the way I've done it with multiple lines.